packages/core/src/plugins/meta/discogs.test.ts

```diff
@@ -0,0 +1,164 @@
+import DiscogsMetaProvider from './discogs';
+import { SimilarArtist } from '../plugins.types';
+import * as SpotifyApi from '../../rest/Spotify';
+
+describe('Tests for DiscogsMetaProvider', () => {
+  const provider = new DiscogsMetaProvider();
+
+  describe('Verify if an artist is on tour', () => {
```

Comment:
I don't think it's needed to unit test small functions like this. Integration tests are more comprehensive, and this function is already included in the output of `fetchArtistDetails`.

Reply:
I tested only the code which I added or had to touch. The expression wrapped in the function was one of the things which caused the issue because of the missing ` lastFmInfo`. So it's most certainly not yet covered by any existing test.

Should I remove the function, the tests and inline the expression? Then my change won't be tested. : )